pkg/reconciler/trigger/controller.go

```diff
@@ -73,16 +73,7 @@ func NewController(
 
 	// Tracker is used to notify us that a Trigger's Broker has changed so that
 	// we can reconcile.
-	r.addressableTracker = addressableInformer.NewTracker(impl.EnqueueKey, controller.GetTrackerLease(ctx))
-	brokerInformer.Informer().AddEventHandler(controller.HandleAll(
-		// Call the tracker's OnChanged method, but we've seen the objects
-		// coming through this path missing TypeMeta, so ensure it is properly
-		// populated.
-		controller.EnsureTypeMeta(
```

Comment:
I don't see where this went.  Are you sure this isn't going to cause problems?

Reply:
It turns out that it was redundant. 

https://github.com/knative/eventing/blob/09dbe44e6c21e970feb1c9a44f48f39dde64929c/pkg/reconciler/trigger/trigger.go#L156

Does the tracking. The deleted section simply did it a second time. To be sure, I tested and verified that the Trigger does correctly reconcile when its Broker changes.